src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx

```diff
@@ -209,6 +210,14 @@ export default class CreateMarketOutcome extends Component {
       updatedMarket.tickSize = value
     }
 
+    // Make sure scalarBigNum, scalarSmallNum, & numTicksBigNum are all BigNumbers
+    if (BigNumber.isBigNumber(scalarBigNum) && BigNumber.isBigNumber(scalarSmallNum) && BigNumber.isBigNumber(numTicksBigNum)) {
```

Comment:
why are we checking that these are big numbers before deciding if they have a proper numTickSize?